packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -91,22 +91,31 @@ export class StatsAnalyzer extends EventsScope {
   successfulCandidatePair: any;
   localIpAddress: string; // Returns the local IP address for diagnostics. this is the local IP of the interface used for the current media connection a host can have many local Ip Addresses
   receiveSlotCallback: ReceiveSlotCallback;
+  isMultistream: boolean;
 
   /**
    * Creates a new instance of StatsAnalyzer
    * @constructor
    * @public
-   * @param {Object} config SDK Configuration Object
-   * @param {Function} receiveSlotCallback Callback used to access receive slots.
-   * @param {Object} networkQualityMonitor class for assessing network characteristics (jitter, packetLoss, latency)
-   * @param {Object} statsResults Default properties for stats
+   * @param {Object} config - SDK Configuration Object
+   * @param {Function} receiveSlotCallback - Callback used to access receive slots.
+   * @param {Object} networkQualityMonitor - Class for assessing network characteristics (jitter, packetLoss, latency)
+   * @param {Object} statsResults - Default properties for stats
+   * @param {boolean | undefined} isMultistream - Param indicating if the media connection is multistream or not
    */
-  constructor(
-    config: any,
-    receiveSlotCallback: ReceiveSlotCallback = () => undefined,
-    networkQualityMonitor: object = {},
-    statsResults: object = defaultStats
-  ) {
+  constructor({
+    config,
+    receiveSlotCallback = () => undefined,
+    networkQualityMonitor = {},
+    statsResults = defaultStats,
+    isMultistream = false,
+  }: {
+    config: any;
+    receiveSlotCallback: ReceiveSlotCallback;
+    networkQualityMonitor: any;
+    statsResults?: any;
+    isMultistream?: boolean;
```

Comment:
Noticed this is optional here. Should we worry about the case in which `isMultistream` is undefined? Then `multistreamEnabled` in the MQE would be undefined as well I think, and I'm not sure if that would fail validation.

Reply:
I use `false` as a fallback here, so far it will always work with either `false` or `true`, you can also notice that `StatsAnalyzer` `isMultistream` works only with `boolean` values.
<img width="452" alt="Screenshot 2024-06-28 at 16 51 31" src="https://github.com/webex/webex-js-sdk/assets/38460776/15bcd9dc-af5c-4138-97c6-61b71b4f73a1">

Reply:
Ah awesome, I didn't notice that. Thanks!